src/api/spec/nyan_unicorn_formatter.rb

```diff
@@ -1,6 +1,29 @@
 # -*- coding: utf-8 -*-
 require 'nyan_cat_formatter'
 
+class RSpec3
```

Comment:
I added this class to overwrite and fix the deprecation warning of nyan-cat-formatter, until the PR is accepted to them. Although the gem looks like abandoned.